helm/alfresco-content-services/values.yaml

```
@@ -477,6 +477,7 @@ alfresco-sync-service:
   # -- Toggle deployment of Alfresco Sync Service (Desktop-Sync)
   # Check [Alfresco Sync Service
   # Documentation](https://github.com/Alfresco/alfresco-helm-charts/tree/main/charts/alfresco-sync-service)
+  replicaCount: 1
   enabled: true
```

```suggestion
  enabled: true
  replicaCount: 1
```

comments are used to generate docs and should be kept consistent